### PR TITLE
tldr: refresh page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/tldr.md [hunk 1]
```diff
@@ -1,20 +1,24 @@
 # tldr
 
-> Displays simple help pages for command-line tools, from the tldr-pages project.
+> Display simple help pages for command-line tools from the tldr-pages project.
 > More information: <https://tldr.sh>.
 
-- Show the tldr page for a command (hint: this is how you got here!):
+- Print a tldr page for a specific command (hint: this is how you got here!):
```
Comment (CleanMachine1) [inline pages/common/tldr.md:6]:
```suggestion
- Print the tldr page for a specific command (hint: this is how you got here!):
```

File: pages/common/tldr.md [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
 
-`tldr {{command}}`
+`tldr {{cd}}`
```
Comment (CleanMachine1) [inline pages/common/tldr.md:8]: I prefer the generic version.

File: pages/common/tldr.md [hunk 3]
```diff
@@ -1,20 +1,24 @@
...
 
-- Show the tldr page for `cd`, overriding the default platform:
+- Print a tldr page for a specific subcommand:
```
Comment (CleanMachine1) [inline pages/common/tldr.md:10]:
```suggestion
- Print the tldr page for a specific subcommand:
```

File: pages/common/tldr.md [hunk 4]
```diff
@@ -1,20 +1,24 @@
...
 
-`tldr -p {{android|linux|osx|sunos|windows}} {{cd}}`
+`tldr {{git}}-{{checkout}}`
 
-- Show the tldr page for a subcommand:
+- Print a tldr page for a command for a specific platform:
```
Comment (CleanMachine1) [inline pages/common/tldr.md:14]:
```suggestion
- Print the tldr page for a command for a specific platform:
```

File: pages/common/tldr.md [hunk 5]
```diff
@@ -1,20 +1,24 @@
...
 
-`tldr {{git-checkout}}`
+`tldr {{-p|--platform}} {{android|linux|osx|sunos|windows}} {{cd}}`
 
-- Update local pages (if the client supports caching):
+- Print a tldr page for a command in a specific language (client may not support this option):
 
-`tldr -u`
+`tldr {{-l|--language}} {{ru}} {{cd}}`
```
Comment (CleanMachine1) [inline pages/common/tldr.md:20]:
```suggestion
`tldr {{-l|--language}} {{ru}} {{command}}`
```

File: pages/common/tldr.md [hunk 6]
```diff
@@ -1,20 +1,24 @@
 # tldr
 
-> Displays simple help pages for command-line tools, from the tldr-pages project.
+> Display simple help pages for command-line tools from the tldr-pages project.
```
Comment (CleanMachine1) [inline pages/common/tldr.md:3]:
```suggestion
> Display simple help pages for command-line tools from the tldr-pages project.
> Different tldr clients do not have the full specification of features, therefore some of these commands may not be accessible to your client
```

File: pages/common/tldr.md [hunk 7]
```diff
@@ -1,20 +1,24 @@
...
-`tldr {{git-checkout}}`
+`tldr {{-p|--platform}} {{android|linux|osx|sunos|windows}} {{cd}}`
 
-- Update local pages (if the client supports caching):
+- Print a tldr page for a command in a specific language (client may not support this option):
 
-`tldr -u`
+`tldr {{-l|--language}} {{ru}} {{cd}}`
+
+- Update local pages (client may not support caching with this option):
```
Comment (CleanMachine1) [inline pages/common/tldr.md:22]:
```suggestion
- Update local pages:
```

File: pages/common/tldr.md [hunk 8]
```diff
@@ -1,20 +1,24 @@
...
+`tldr {{git}}-{{checkout}}`
 
-- Show the tldr page for a subcommand:
+- Print a tldr page for a command for a specific platform:
 
-`tldr {{git-checkout}}`
+`tldr {{-p|--platform}} {{android|linux|osx|sunos|windows}} {{cd}}`
 
-- Update local pages (if the client supports caching):
+- Print a tldr page for a command in a specific language (client may not support this option):
```
Comment (CleanMachine1) [inline pages/common/tldr.md:18]:
```suggestion
- Print a tldr page for a command in a specific language:
```

File: pages/common/tldr.md [hunk 9]
```diff
@@ -1,20 +1,24 @@
...
+`tldr {{git}}-{{checkout}}`
 
-- Show the tldr page for a subcommand:
+- Print a tldr page for a command for a specific platform:
 
-`tldr {{git-checkout}}`
+`tldr {{-p|--platform}} {{android|linux|osx|sunos|windows}} {{cd}}`
 
-- Update local pages (if the client supports caching):
+- Print a tldr page for a command in a specific language:
```
Comment (CleanMachine1) [inline pages/common/tldr.md:18]:
```suggestion
- Print the tldr page for a command in a specific language:
```

File: pages/common/tldr.md [hunk 10]
```diff
@@ -1,20 +1,25 @@
...
-`tldr {{git-checkout}}`
+`tldr {{-p|--platform}} {{android|linux|osx|sunos|windows}} {{command}}`
 
-- Update local pages (if the client supports caching):
+- Print the tldr page for a command in a specific language:
 
-`tldr -u`
+`tldr {{-l|--language}} {{ru}} {{command}}`
+
+- Update local pages:
```
Comment (CleanMachine1) [inline pages/common/tldr.md:23]: We cannot add this since it is not required in our spec
```suggestion
```

File: pages/common/tldr.md [hunk 11]
```diff
@@ -1,20 +1,25 @@
 # tldr
 
-> Displays simple help pages for command-line tools, from the tldr-pages project.
+> Display simple help pages for command-line tools from the tldr-pages project.
+> Different tldr clients do not have the full specification of features, therefore some of these commands may not be accessible to your client.
```
Comment (CleanMachine1) [inline pages/common/tldr.md:4]:
```suggestion

```

**Review by CleanMachine1 — APPROVED** [review]

File: pages/common/tldr.md [hunk 12]
```diff
@@ -1,20 +1,20 @@
...
+- Print the tldr page for a specific subcommand:
 
-`tldr -p {{android|linux|osx|sunos|windows}} {{cd}}`
+`tldr {{command}}-{{subcommand}}`
 
-- Show the tldr page for a subcommand:
+- Print the tldr page for a command for a specific platform:
 
-`tldr {{git-checkout}}`
+`tldr {{-p|--platform}} {{android|linux|osx|sunos|windows}} {{command}}`
```
Comment (navarroaxel) [inline pages/common/tldr.md:16]: The client specification says:
> Clients MAY choose to only implement the short version of an option, ignoring the long form.
So:
```suggestion
`tldr {{-p}} {{android|linux|osx|sunos|windows}} {{command}}`
```

File: pages/common/tldr.md [hunk 13]
```diff
@@ -1,20 +1,20 @@
...
 
-- Update local pages (if the client supports caching):
+- Print the tldr page for a command in a specific language:
```
Comment (navarroaxel) [inline pages/common/tldr.md:18]: but maybe the client doesn't implement cache :man_shrugging:

Comment (CleanMachine1) [reply pages/common/tldr.md:18]: also this description doesn't match the command at all

File: pages/common/tldr.md [hunk 14]
```diff
@@ -1,20 +1,20 @@
...
+- Print the tldr page for a command for a specific platform:
 
-`tldr {{git-checkout}}`
+`tldr {{-p}} {{android|linux|osx|sunos|windows}} {{command}}`
 
-- Update local pages (if the client supports caching):
+- Update the cache of tldr pages:
 
-`tldr -u`
+`tldr {{-u|--update}}`
```
Comment (CleanMachine1) [inline pages/common/tldr.md:20]:
```suggestion
`tldr {{-u}}`
```

File: pages/common/tldr.md [hunk 15]
```diff
@@ -1,20 +1,20 @@
...
+- Print the tldr page for a specific subcommand:
 
-`tldr -p {{android|linux|osx|sunos|windows}} {{cd}}`
+`tldr {{command}}-{{subcommand}}`
 
-- Show the tldr page for a subcommand:
+- Print the tldr page for a command for a specific [p]latform:
 
-`tldr {{git-checkout}}`
+`tldr {{-p}} {{android|linux|osx|sunos|windows}} {{command}}`
```
Comment (marchersimon) [inline pages/common/tldr.md:16]: Since when do we put flags in placeholder syntax?
```suggestion
`tldr -p {{android|linux|osx|sunos|windows}} {{command}}`
```

File: pages/common/tldr.md [hunk 16]
```diff
@@ -1,20 +1,20 @@
...
 
-- Update local pages (if the client supports caching):
+- [u]pdate the cache of tldr pages:
```
Comment (marchersimon) [inline pages/common/tldr.md:18]:
```suggestion
- [u]pdate the local cache of tldr pages:
```

File: pages/common/tldr.md [hunk 17]
```diff
@@ -1,20 +1,20 @@
...
 
-`tldr -u`
+`tldr {{-u}}`
```
Comment (marchersimon) [inline pages/common/tldr.md:20]:
```suggestion
`tldr -u`
```

File: pages/common/tldr.md (hunk 15 above)
Comment (EmilySeville7cfg) [reply pages/common/tldr.md:16]: It's my mistake here. Thanks! :)

**Comment by marchersimon** [comment]
Merging, since all comments were addressed and the missing reviews were request quite some while ago
